features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/js/applications.js

```diff
@@ -630,8 +637,8 @@ $("#application-actions").each(function(){
         $('#messageModal a.btn-other').html(i18n.t("No"));
         $('#messageModal a.btn-primary').click(function() {
             jagg.post("/site/blocks/application/application-remove/ajax/application-remove.jag", {
-                action:"removeApplication",
-                application:appName
+                action:"removeApplicationById",
+                applicationId:appId
```

Comment:
to make it consistent shall we change the param applicationId to appId?

Reply:
Fixed with 89b6956